examples/dt/uni05epsilon/edpm-pre-ceph/nodeset/values.yaml.org

```diff
@@ -0,0 +1,156 @@
+---
```

Comment:
Is this a leftover/backup file?

Reply:
A rebase artifact, thanks for catching! :heart:  Removing